BankSystem.cpp

```diff
@@ -0,0 +1,79 @@
+#include <iostream>
+
+using namespace std;
+Fun(int Ac1, int Ac2, int acc1, int acc2,int & Ac1B,int & Ac2B) {
+    int am1;
+    cout << "Enter The Account Number You Want To Withdraw Money From " << endl;
+    cin >> acc1;
+    if (acc1 != Ac1 && acc1 != Ac2) { cout << "Invalid Account Number " << endl; }
+    cout << "Enter The Account Number You Want To Add Money To " << endl;
+    cin >> acc2;
+    if (acc2 != Ac1 && acc2 != Ac2) { cout << "Invalid Account Number " << endl; }
+    cout << "Enter The Amount Of Money  " << endl;
+    cin >> am1;
+    if (acc1 == Ac1) {
```

Comment:
The amount entered may be greater than the account from which money is to be transferred
(no if statement to check that )